_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -87,3 +87,28 @@ We highly recommend reviewing the detailed list of breaking changes in our docum
 We advise all Solidity developers to upgrade to version 0.8.18 to take advantage of these improvements and optimizations. Our team remains dedicated to continuously enhancing Solidity to benefit the entire Ethereum ecosystem.
 
 Get ready to unleash your creativity and build impactful decentralized applications with the latest version of the Solidity Compiler. Happy coding!
+
+## UPDATE 2022-02-02: Rebuilt Linux binary for Solidity 0.8.18
+
+The Linux binary for the 0.8.18 release was originally built against a too recent version of glibc and thereby did not run on older Linux systems. 
+Therefore, we rebuilt and replaced the binary so that you may encounter an older version of ``solc-linux-amd64-v0.8.18+commit.87f61d96`` that does not match the hash you can find on the list of [currently published Linux binaries](binaries.soliditylang.org/linux-amd64/list.json).
```

Comment:
The link is broken because I did not include the `https://` part:
```suggestion
Therefore, we rebuilt and replaced the binary so that you may encounter an older version of ``solc-linux-amd64-v0.8.18+commit.87f61d96`` that does not match the hash you can find on the [list of currently published Linux binaries](https://binaries.soliditylang.org/linux-amd64/list.json).
```